include/Network/HTTP/HTTPException.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Throwable exceptions to return HTTP status code
+ *
+ * This list of Exception has be extracted from
+ * here http://racksburg.com/choosing-an-http-status-code/
+ */
+namespace Friendica\Network\HTTP;
+
+class HTTPException extends \Exception {
```

Comment:
STANDARDS: please add a doxygen header for the class. If you don't want to be creative you can just copy the doxygen header of the file (without  `@file` line).
It is just for having the description of the class when the doxygen docu is created